tests/conftest.py

```diff
@@ -49,7 +48,7 @@ def pytest_collection_modifyitems(items) -> None:
             # we raise an exception for all other dirs that are passed in
             if item_path == top_doctest_dir:
                 item.add_marker("doctest")
-            elif str(item_path).startswith(str(modin_doctest_dir)):
+            elif "modin" in str(item_path):
```

Comment:
are these changes from a different PR/ticket?

Reply:
I rebased, this should be solved now